src/ztc/api/tests/test_informatieobjecttype.py

```diff
@@ -174,7 +174,7 @@ def test_create_informatieobjecttype_fail_not_unique(self):
     def test_publish_informatieobjecttype(self):
         informatieobjecttype = InformatieObjectTypeFactory.create()
         informatieobjecttypee_url = get_operation_url(
-            "informatieobjecttype_publish", uuid=informatieobjecttype.uuid
+            "informatieobjecttypen_publish", uuid=informatieobjecttype.uuid
```

Comment:
Deze zou ook enkelvoudig moeten zijn, wellicht dat er hier een aanpassing voor nodig is `vng-api-common`